endpoint-deprecation/service_endpoint_requests/vis.json

```diff
@@ -0,0 +1,64 @@
+{
+  "variables": [
+      {
+          "name": "start_time",
+          "type": "PX_STRING",
+          "description": "The relative start time of the window. Current time is assumed to be now",
+          "defaultValue": "-5m"
+      },
+      {
+          "name": "service",
+          "type": "PX_SERVICE",
+          "description": "The name of the service to get stats for.",
+          "defaultValue": "default"
+      },
+      {
+          "name": "endpoint",
+          "type": "PX_STRING",
+          "description": "The endpoint to look at stats for. Format: /path/to/endpoint/*/more/path ",
+          "defaultValue": "/"
```

Comment:
not sure this should be / , don't we have some that have no path aka empty string? we can also require an arg here and drop defaultValue altogether